src/client/manager/provider.js

```diff
@@ -4,6 +4,7 @@ import React from 'react';
 import { Provider } from '@kadira/storybook-ui';
 import addons from '@kadira/storybook-addons';
 import createChannel from '@kadira/storybook-channel-pagebus';
+import createDatabase from '@kadira/storybook-database-local';
```

Comment:
Do we need to move this to another repo? If there's a reason we can move the server code too. (I mean the middleware)